src/steps/check-pardot-tracking.ts

```diff
@@ -32,7 +32,7 @@ export class CheckPardotTrackingStep extends BaseStep implements StepInterface {
     const stepData: any = step.getData().toJavaScript();
     const aid = stepData.aid;
     const cid = stepData.cid;
-    const customDomain = stepData.customDomain || 'https://pi.pardot.com';
+    const customDomain = stepData.customDomain || 'http://pi.pardot.com';
```

Comment:
@saludangelito, the protocol is going to vary based on the URL being navigated to (Pardot seems to match the protocol of the page loaded, for whatever reason).  Example:

```yml
- step: When I navigate to http://go.pardot.com/l/811013/2019-10-16/3mxyx
- step: Then the tracking code for pardot account 812013 and campaign 45363 should have loaded
- step: And when I navigate to https://go.pardot.com/l/811013/2019-10-16/3mxyx
- step: Then the tracking code for pardot account 812013 and campaign 45363 should have loaded
```

A couple of possible solutions:
- Update the protocol of `customDomain` based on the protocol of the current URL.
- Or update the underlying `getNetworkRequests()` method to ignore protocol (e.g. parse the URL and compare the host values, rather than doing a string check that includes protocol + host)

Reply:
should i also update the user provided customDomain?

Reply:
Nah, I think we can take the user input at face-value.